learning.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: page
+permalink: /learning
+title: Berlin Tech Worker Coalition Learning Group
```

Comment:
I'd prefer `Berlin Learning Group`, even in desktop, it doesn't fit the whole screen. See screenshot below 


![Screen Shot 2019-07-30 at 19 17 11](https://user-images.githubusercontent.com/7111514/62150744-0b72fa80-b2ff-11e9-894a-d6970d06f271.png)

Reply:
totally right